test/csuite/schema_abort/main.c

```diff
@@ -312,6 +311,20 @@ test_cursor(THREAD_DATA *td)
     testutil_check(session->close(session, NULL));
 }
 
+/*
+ * set_flush_tier_delay --
+ *     Set up a random delay for the next flush_tier.
+ */
+static void
+set_flush_tier_delay(WT_RAND_STATE *rnd)
+{
+    /*
+     * We are checkpointing constantly, and we'll do a flush tier with a random delay between 0 -
```

Comment:
The "We are checkpointing constantly" part of the comment is not correct.  And I think it's more helpful to describe the intention of the code in terms easier to understand.  How about: "We are checkpointing randomly every 0-3 seconds, and we'll do a flush tier randomly every 0-10 seconds".

Reply:
I would avoid mentioning the checkpoint interval in the comment section because if someone updates the time of `MAX_CKPT_INVL` then the comment also needs to be updated. How about  "We are checkpointing randomly every MAX_CKPT_INVL seconds, and we'll do a flush tier randomly every 0-10 seconds" ?

Reply:
A slight modification?  "We are checkpointing with a random interval up to MAX_CKPT_INVL seconds, and...."